dvc/scm/tree.py

```diff
@@ -37,6 +37,9 @@ def walk_files(self, top):
                 # NOTE: os.path.join is ~5.5 times slower
                 yield f"{root}{os.sep}{file}"
 
+    def makedirs(self, path, mode=0o777, exist_ok=True):
+        pass
+
```

Comment:
Really need to consolidate all this mess https://github.com/iterative/dvc/issues/4050 .